models/text-generation--opencsg--opencsg-deepseek-coder-1.3b-v0.1-pipeline.yaml

```diff
@@ -0,0 +1,47 @@
+deployment_config:
+  autoscaling_config:
+    min_replicas: 1
+    initial_replicas: 1
+    max_replicas: 3
+    target_num_ongoing_requests_per_replica: 1.0
+    metrics_interval_s: 10.0
+    look_back_period_s: 30.0
+    smoothing_factor: 1.0
+    downscale_delay_s: 300.0
+    upscale_delay_s: 90.0
+  ray_actor_options:
+    num_cpus: 0.5 # for a model deployment, we have 3 actor created, 1 and 2 will cost 0.1 cpu, and the model infrenece will cost 6(see the setting in the end of the file)
+model_config:
+  warmup: False
+  model_task: text-generation
+  model_id: OpenCSG/opencsg-deepseek-coder-1.3b-v0.1
```

Comment:
Which repository the `mode_id` refer to?  huggingface or csghub

Reply:
BTW, why we need to support both `defaulttransformers` and `default `? is the behavior different?

Reply:
Right, just default is ok.